doc/quick-start/qs-guide.rst

```diff
@@ -232,7 +232,149 @@ If all the steps were followed correctly, it will be possible to see that the KN
+
+Interacting with Application through KNoT Cloud SDK
+---------------------------------------------------
+
+#. Create a new directory and start a NodeJS application on it.
+
+    .. code-block:: bash
+
+      npm init -y
+
+#. Install the KNoT Cloud SDK for JavaScript.
+
+    .. code-block:: bash
+
+      npm i -s @cesarbr/knot-cloud-sdk-js
+
+#. Create a ``index.js`` file and import the ``knot-cloud-sdk-js``.
+
+   .. code-block:: javascript
+
+      const { Client } = require('@cesarbr/knot-cloud-sdk-js');
+
+#. Create a client connection instance with the KNoT Cloud WebSocket server by using the application credentials that you have obtained above.
+
+   .. code-block:: javascript
+
+    const client = new Client({
+      hostname: 'ws.knot.cloud',
+      protocol: 'wss',
+      port: 443,
+      pathname: '/ws',
+      id: '3c92790f-f265-46c9-bbf8-e440f0447587', // APP ID
+      token: '826faa7d545e39c8b2a198c74d0da54f95dfea55', // APP TOKEN
+    });
+
+#. Send ``setData`` command to turn off the KNoT Dongle's LED (you can get it's ID from the gateway interface).
+
+   .. code-block:: javascript
+
+      const data = [
+        {
+          sensorId: 0, // LED's sensorID
+          value: false, // New LED's value
+        },
+      ];
+
+      client.on('ready', () => {
+        client.setData('2828b4c983f2d9d1', data); // Send setData command, passing to it the KNoT Dongle's ID and the data.
+      });
+
+      client.on('sent', () => {
+        client.close(); // close the connection after command is sent
+      });
+
+      client.on('error', (err) => {
+        console.log(err);
+        console.log('Connection refused');
+      });
+
+      client.connect();
+
+#. Listen to data events sent by the KNoT Dongle. These events can be listened to by registering a handler with ``on('data')``.
+
+   .. note:: This event listener will receive every data events sent by all things associated with you user. To filter them, you just need to compare the ``from`` field with the KNoT Thing ID you want to listen.
+
+   .. code-block:: javascript
+
+    client.on('ready', () => {});
+
+    client.on('data', (data) => {
+      if (data.from === '2828b4c983f2d9d1') {
+        console.log(JSON.stringify(data, null, 2));
+      }
+    })
+
+    client.on('error', (err) => {
+      console.log(err);
+      console.log('Connection refused');
+    });
+
+    client.connect();
+
+    // Expected data object
+    // {
+    //  from: '2828b4c983f2d9d1',
+    //  payload: {
+    //    sensorId: 0,
+    //    value: false,
+    //  },
+    // }
+
+#. Run the example.
+
+  .. code-block:: bash
+
+    NODE_TLS_REJECT_UNAUTHORIZED=0 node index.js
+
+  .. note:: The environment variable `NODE_TLS_REJECT_UNAUTHORIZED <https://nodejs.org/api/cli.html#cli_node_tls_reject_unauthorized_value>`_ need to be set to '0' in order to disable the TLS certificate verification, since you are connecting to a WebSocket Secure server. It should be used only on development.
```

Comment:
Then what should be used in non-development environments? Is there any helpful link for the user to read in this case?

Reply:
This line is too long. Could you, please, break it?

Reply:
In non-development environments, we don't need to set this variable. I think "It should be used only on development" is clear enough. Do you have some suggestions to improve it?

Reply:
Please, have a maximum line length of 79 characters. As described by https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html

Reply:
Done.